conf/messages.cy

```diff
@@ -562,7 +562,8 @@ error.agentProperties.filter=Rhaid i chi nodi rhywbeth i chwilio amdano
 
 #Manage Agent
 propertyRepresentation.manageAgent.title=Rheoli asiant
-propertyRepresentation.manageAgent.question=Beth ydych chi eisiau ei wneud i’r asiant {0}?
+propertyRepresentation.manageAgent.question=What do you want to do with your agent {0}?

```

Comment:
Are we waiting on translations for line 565, 593 & 594?

Reply:
Yes. This ticket hasn't got translation yet